src/hooks/useURLParams.ts

```diff
@@ -0,0 +1,27 @@
+import { useState, useMemo } from 'react'
+
+function setURLFilterParams(filterString: string, moveNow?: boolean): string | void | boolean {
+  if (!window || !window.history) return false
+  const defaultLocation = window.location.href.split('?')[0]
+  if (!filterString) return null
+  return moveNow
+    ? (window.location.href = `${defaultLocation}${filterString}`)
+    : window.history.replaceState(null, null, `?${filterString}`)
+}
+
+export function useLocation(): URLSearchParams {
+  const [search, setSearch] = useState(location.search)
+
+  useMemo(() => {
+    search ? console.debug('SEARCH: ', search) : console.debug('No search params')
```

Comment:
do we really want to log this in the console?

Reply:
ah nope, artifact. will remove